treeherder/webapp/api/urls.py

```diff
@@ -139,4 +139,5 @@ class TextLogErrorRouter(routers.DefaultRouter):
     url(r'^failures/$', intermittents_view.Failures.as_view(), name='failures'),
     url(r'^failuresbybug/$', intermittents_view.FailuresByBug.as_view(), name='failures-by-bug'),
     url(r'^failurecount/$', intermittents_view.FailureCount.as_view(), name='failure-count'),
+    url(r'^perfbyrevision/$', performance_data.PerformanceByRevision.as_view(), name='perf-by-revision'),
```

Comment:
A few things here:

1. I think this should go under `/performance/` (see other examples of this above, e.g. performance-issue-tracker)
2. As above, I think associating this with revision is not quite right. How about `/performance/data/summary/`?

Reply:
Data seems redundant. Changed to `performance/summary/`.

Reply:
wfm